sw/device/lib/dif/dif_rv_core_ibex.c

```diff
@@ -205,3 +205,61 @@ dif_result_t dif_rv_core_ibex_clear_error_status(
 
   return kDifOk;
 }
+
+dif_result_t dif_rv_core_ibex_enable_nmi(const dif_rv_core_ibex_t *rv_core_ibex,
+                                         bool alert, bool wdog) {
```

Comment:
maybe the API for this should be slightly different. feels a bit odd to return `kDifBadArg` when `alert` or `wdog` is false. Maybe this should be something like:

```suggestion
typdef enum dif_rv_core_ibex_nmi_source {
  kDifRvCoreIbexNmiSourceAlert = 0,
  kDifRvCoreIbexNmiSourceWdog = 0,
} dif_rv_core_ibex_nmi_source_t

dif_result_t dif_rv_core_ibex_enable_nmi(const dif_rv_core_ibex_t *rv_core_ibex,
                                         dif_rv_core_ibex_nmi_source_t nmi_source) {
```

wdyt?

Reply:
You are right. So I changed the enable and clear API.